packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -66,6 +66,170 @@ return spark.meetings
+  .withMedia() // add the media type, to use the media peer connection, can also add on withScreen() for screen peer connection
+  .build() // finish up building the getStats instance and get a reference back
+  .getMedia() // get the media instance off of getStats
+  .withId() // add an ID to the media type of stats, optional
+  .withEvents() // trigger 'stats:update' events on media, see 'Events' below, optional
+  .withHistory() // collect history with DEFAULT_HISTORY_MAX size (default is 30 minutes, 1800 entries), optional
+  .withAggregator() // aggregate data from the stream each time, to have a summary of each stat, optional
+  .withInterval() // find the actual values between each registered interval, optional
+  .withFilter(meeting.mediaPeerConnection) // create the Transform stream filter on the media instance to convert the data to
+  // a nice human readable format, and parse out relevant stats, defined by DEFAULT_TRANSFORM_REGEX, see nodejs Transform Stream
+  .build() // finish up building the media stats instance and get a reference back
+  .onData((filtered) => { // do something with each filtered data beyond any of the added options, pass a callback
+    console.log(filtered);
+  });
+// OR
+// for raw get stats without a filter, getting just a Readable stream back, see nodejs Readable Stream
+// not for use with events, history, aggregator, interval, will result in undefined behavior
+meeting
+  .getStats() // get the stats instance, creates a new one if one does not exist on the meeting object
+  .withMedia() // add the media type, to use the media peer connection, can also add on withScreen() for screen peer connection
+  .build() // finish up building the getStats instance and get a reference back
+  .getMedia() // get the media instance off of getStats
+  .withStream(meeting.mediaPeerConnection); // create the Readable stream on the media instance to be able to set up a data pipe
+  // or a simple listener for the stats reports coming back, to do your own stats filtering/analysis
+meeting.getStats().getMedia().getStream().on('data', (data) => {
+  // do something with the raw getStats report
+});
+// afterwards for each you may just call to get a stats instance, without having to create it's configuration using with()
+meeting.getStats()...
+```
+
+##### Options
+```javascript
+// functionally equivalent to the builder
+// pass in with a standard options object which pieces you want to create on the stats instance
+meeting.getStats({ // create the stats reference
+  screen: { // with screen option
+    id: true, // with an id on the screen stats
+    history: true, // with history
+    interval: true, // with interval
+    filter: meeting.screenPeerConnection, // with filter
+    events: true // with events
+  },
+  media: { // can add media or exclude
+    ...
+  }
+});
+meeting.getStats().getScreen().onData((filtered) => {
```

Comment:
Could these methods be chained? Or do you have to call `meeting.getStats()` twice?

Reply:
these methods could be chained